pkg/fanal/image/daemon/docker.go

```diff
@@ -11,10 +11,17 @@ import (
 
 // DockerImage implements v1.Image by extending daemon.Image.
 // The caller must call cleanup() to remove a temporary file.
-func DockerImage(ref name.Reference) (Image, func(), error) {
+func DockerImage(ref name.Reference, host string) (Image, func(), error) {
 	cleanup := func() {}
 
-	c, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	var c *client.Client
```

Comment:
```golang
    opts:=[]client.Opt{client.FromEnv, client.WithAPIVersionNegotiation()} 
    if host != "" {
	// adding host parameter to the last assuming it will pickup more preference
	opts=append(opts, client.WithHost(host))
    }

    c, err:= client.NewClientWithOpts(opts...)
```

Reply:
✅